backend/src/domain/models/Game.js

```diff
@@ -25,6 +25,7 @@ export class Game {
    * 0 represents the start of the game (calling nextRound)
    */
   async startCountdown() {
+    // let all users in game start countdown
```

Comment:
Maybe start these comments with a capital letter, to keep it consistent with the rest of the project. :)

Reply:
ignore the emoji, it wont go away